packages/amplify-codegen-e2e-core/src/categories/api.ts

```diff
@@ -49,9 +49,7 @@ export function addApiWithoutSchema(cwd: string, opts: Partial<AddApiOptions & {
       .sendCarriageReturn()
       .wait('Do you want to edit the schema now?')
       .sendConfirmNo()
-      .wait(
-        '"amplify publish" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud',
-      )
+      .wait('"amplify publish" will build all your local backend and frontend resources')
```

Comment:
Strange error.

```
● codegen remove tests - iOS › Does not delete files during codegen remove
--
351 |  
352 | Expected value   "\"amplify publish\" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud"
353 | Received:
354 | "ision it in the cloud"
355 |  
356 | Message:
357 | expected "ision it in the cloud" to contain "\"amplify publish\" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud"
358 |  
359 | Difference:
360 |  
361 | - Expected
362 | + Received
363 |  
364 | - "amplify publish" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud
365 | + ision it in the cloud
```

The message is correct when running manually. Maybe there is a maximum length? The "v" in "provision" is the 120th character.

This change was also made on the CLI E2E when enabling windows.
https://github.com/aws-amplify/amplify-cli/commit/97c6de42f906d56a5fd0d78c4827110508955b93

Reply:
Alternatively, you can wrap the message with `*/` regex to ignore subsequent parts. Just FYI.